chronicle/Models/FullQualifiedName.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FullQualifiedName.swift
+//  FullQualifiedName
+//
+//  Created by Alfonce Nzioka on 11/3/21.
+//
+
+import Foundation
+
+struct FullQualifiedName: Codable, Hashable {
+    let namespace: String
+    let name: String
+    
+    init(namespace: String, name: String) {
+        self.namespace = namespace
+        self.name = name
+    }
+}
+
+extension FullQualifiedName {
```

Comment:
shouldn't this be `FullyQualifiedName`?

Reply:
No, it's fine as it is.

Trying to emulate the same class we use in our edm. 
(https://olingo.apache.org/javadoc/odata4/org/apache/olingo/commons/api/edm/FullQualifiedName.html#FullQualifiedName-java.lang.String-)